src/main/java/com/github/fppt/jedismock/operations/RO_zrange.java

```diff
@@ -0,0 +1,59 @@
+package com.github.fppt.jedismock.operations;
+
+
+import com.github.fppt.jedismock.server.Response;
+import com.github.fppt.jedismock.server.Slice;
+import com.github.fppt.jedismock.storage.RedisBase;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.Sets;
+
+import java.util.Iterator;
+import java.util.List;
+import java.util.Set;
+
+import static com.github.fppt.jedismock.Utils.convertToInteger;
+import static com.github.fppt.jedismock.Utils.deserializeObject;
+
+public class RO_zrange extends AbstractRedisOperation {
```

Comment:
I am wondering if we might be able to abstract this into a common `RO_range` class. That way `RO_zrange` and `RO_lrange` might be able to get rid of some bloat. If you look at `RO_lrange` it almost does the same thing as this.